packages/sdk-router/src/typechain/FastBridgeInterceptor.ts

```
@@ -0,0 +1,206 @@
+    MAX_ORIGIN_AMOUNT_DIFF(overrides?: CallOverrides): Promise<BigNumber>
+
+    NATIVE_GAS_TOKEN(overrides?: CallOverrides): Promise<string>
+
+    bridgeWithInterception(
+      params: IFastBridge.BridgeParamsStruct,
+      interceptorParams: IFastBridgeInterceptor.InterceptorParamsStruct,
+      overrides?: CallOverrides
+    ): Promise<void>
+  }
+
+  filters: {}
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)